<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1471</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CareerJob-Professor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Source/6995985.png</normaloff>../Source/6995985.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #22222e;</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>331</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #2f55a7;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>811</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(55, 98, 199);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>D:/lemon/Eye_Pasports/Без имени-1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>BankGothic Md BT</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

background-color: none;</string>
    </property>
    <property name="text">
     <string>Professor</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: #22222e;
border: 2px solid #2f55a7;
border-radius: 20;
color:white
}

QLineEdit:focus {
	background-color: rgb(85, 85, 255);
	

}</string>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Введите код направления</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="process_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #22222e;
border: 2px solid #2f55a7;
border-radius: 20;
color:white
}

QPushButton:hover {
	background-color: rgb(85, 85, 255);
	

}</string>
    </property>
    <property name="text">
     <string>Найти компетенции</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Source/icons8-поиск-58.png</normaloff>../Source/icons8-поиск-58.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(55, 98, 199);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Source/1491254409-leftarrowbackwardsign_82959.png</normaloff>../Source/1491254409-leftarrowbackwardsign_82959.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="process_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>730</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #22222e;
border: 2px solid #2f55a7;
border-radius: 20;
color:white
}

QPushButton:hover {
	background-color: rgb(85, 85, 255);
	

}</string>
    </property>
    <property name="text">
     <string>Регистрация студента</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Source/icons8-студент-100.png</normaloff>../Source/icons8-студент-100.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#comboBox{
border: 2px solid #2f55a7;
border-radius: 20px;
padding-left: 10px;
color: white;
}


QComboBox QAbstractItemView {
  color: rgb(85, 170, 255);	
  background-color: #373e4e;
  padding: 10px;
	font-size: 10px;
  selection-background-color: rgb(39, 44, 54);
outline:0px;

}

QComboBox QAbstractItemView ::item{
height:50px;

}



#comboBox::drop-down{
border:0px;

}

#comboBox:on {
border: 4px solid  rgb(85, 85, 255);

}

QComboBox QListView
{
    border-style: none;

 
}

#comboBox QListView:item:hover{
background-color ;
}

#comboBox QListView:item:selected
{
background-color:white;
}

</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: none;</string>
    </property>
    <property name="text">
     <string>Выбрать все</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: none;</string>
    </property>
    <property name="text">
     <string>Поиск по коду</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: none;</string>
    </property>
    <property name="text">
     <string>Поиск по направлению</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>531</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;

</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>30</y>
      <width>491</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
color: rgb(255, 255, 255);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process_4">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>710</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #22222e;
border: 2px solid #2f55a7;
border-radius: 20;
color:white
}

QPushButton:hover {
	background-color: rgb(85, 85, 255);
	

}</string>
    </property>
    <property name="text">
     <string>Поиск вакансий</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Source/icons8-новая-вакансия-64.png</normaloff>../Source/icons8-новая-вакансия-64.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="process_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>710</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #22222e;
border: 2px solid #2f55a7;
border-radius: 20;
color:white
}

QPushButton:hover {
	background-color: rgb(85, 85, 255);
	

}</string>
    </property>
    <property name="text">
     <string>Статистика по вакансиям</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Source/icons8-статистика-100.png</normaloff>../Source/icons8-статистика-100.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: none;
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Source/icons8-крестик-78.png</normaloff>../Source/icons8-крестик-78.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit</zorder>
   <zorder>listWidget</zorder>
   <zorder>process_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>process_3</zorder>
   <zorder>comboBox</zorder>
   <zorder>checkBox</zorder>
   <zorder>radioButton</zorder>
   <zorder>radioButton_2</zorder>
   <zorder>graphicsView</zorder>
   <zorder>listWidget_2</zorder>
   <zorder>process_4</zorder>
   <zorder>process_5</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
